packages/common/src/controllers/submissionController.ts

```diff
@@ -16,55 +16,57 @@ const controller = (dependencies: Dependencies) => {
 			try {
 				const categoryId = Number(req.params.categoryId);
 				const files = req.files as Express.Multer.File[];
```

Comment:
this is previously handled by the router(submissionRouter.ts). Where we're using `Multer` array function to populate this as an array of Files.

Reply:
rather than asserting type here, this line could be abstracted to a separate function, wherein we could add validation for the files (?), and then return with the appropriate type